template/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=de.data_experts.profil.ceres.nuve.asyncapi.Config
```

Comment:
What is it?

Reply:
Spring does normaly not search libraryies (jars in the build path) for beans and components.
To tell a spring application to load beans and components from an external jar, this jar needs a Config class that is specified in a spring.factories file.

More on this here:
https://docs.spring.io/autorepo/docs/spring-boot/2.0.0.M3/reference/html/boot-features-developing-auto-configuration.html